django_app/settings.py

```diff
@@ -225,3 +225,9 @@
 # https://docs.djangoproject.com/en/4.1/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+
```

Comment:
A comment? Cuz no one knows what this stuff is or why it's here if they are new to the project?

Reply:
Sure. Will be adding that